website/content/docs/enterprise/audit/exclusion.mdx

```diff
@@ -0,0 +1,285 @@
+
+### Request audit entry
+
+```json
+{
+  "auth": <auth>,
+  "error": "",
+  "forwarded_from": "",
+  "request": <request>,
+  "time": "",
+  "type": ""
+}
+```
+
+### Response audit entry
+
+```json
+{
+  "auth": <auth>,
+  "error": "",
+  "forwarded_from": "",
+  "request": <request>,
+  "response": <response>,
+  "time": "",
+  "type": ""
+}
+```
+
+### Auth object (`<auth>`)
+
+Complete with example data to show simple types (`string`, `bool`, `int`), other
+JSON examples which refer to `<auth>` are referring to this object.
+
+```json
+{
+  "accessor": "",
+  "client_token": "",
+  "display_name": "",
+  "entity_created": "",
+  "entity_id": "",
+  "external_namespace_policies": {
+    "allowed": true,
+    "granting_policies": [
+      {
+        "name": "",
+        "namespace_id": "",
+        "namespace_path": "",
+        "type": ""
+      }
+    ]
+  },
+  "identity_policies": [
+    ""
+  ],
+  "metadata": {},
+  "no_default_policy": false,
+  "num_uses": 10,
+  "policies": [
+    ""
+  ],
+  "policy_results": {
+    "allowed": true,
+    "granting_policies": [
+      {
+        "name": "",
+        "namespace_id": "",
+        "namespace_path": "",
+        "type": ""
+      }
+    ]
+  },
+  "remaining_uses": 5,
+  "token_policies": [
+    ""
+  ],
+  "token_issue_time": "",
+  "token_ttl": 3600,
+  "token_type": ""
+}
+```
+
+### Request object (`<request>`)
+
+Complete with example data to show simple types (`string`, `bool`, `int`), other
+JSON examples which refer to `<request>` are referring to this object.
+
+```json
+{
+  "client_certificate_serial_number": "",
+  "client_id": "",
+  "client_token": "",
+  "client_token_accessor": "",
+  "data": {},
+  "id": "",
+  "headers": {},
+  "mount_accessor": "",
+  "mount_class": "",
+  "mount_point": "",
+  "mount_type": "",
+  "mount_running_version": "",
+  "mount_running_sha256": "",
+  "mount_is_external_plugin": "",
+  "namespace": {
+    "id": "",
+    "path": ""
+  },
+  "operation": "",
+  "path": "",
+  "policy_override": true,
+  "remote_address": "",
+  "remote_port": 1234,
+  "replication_cluster": "",
+  "request_uri": "",
+  "wrap_ttl": 60
+}
+```
+
+### Response object (`<response>`)
+
+Complete with example data to show simple types (`string`, `bool`, `int`), other
+JSON examples which refer to `<response>` are referring to this object.
+
+```json
+{
+  "auth": <auth>,
+  "data": {},
+  "headers": {},
+  "mount_accessor": "",
+  "mount_class": "",
+  "mount_is_external_plugin": false,
+  "mount_point": "",
+  "mount_running_sha256": "",
+  "mount_running_plugin_version": "",
+  "mount_type": "",
+  "redirect": "",
+  "secret": {
+    "lease_id": ""
+  },
+  "wrap_info": {
+    "accessor": "",
+    "creation_path": "",
+    "creation_time": "",
+    "token": "",
+    "ttl": 60,
+    "wrapped_accessor": ""
+  },
+  "warnings": [
+    ""
+  ]
+}
+```
+## JSON schema
+
+The JSON schema for both request and response audit entries are included below.
```

Comment:
```suggestion
```

I don't think the section heading is helping here. I'd recommend removing it and bumping the request/response sections to `h2` so they show in the TOC